03-task-manager/starter/QuizAnswers2.txt

```diff
@@ -0,0 +1,26 @@
+1.) In this lesson, you created a middleware function called 
+asyncWrapper. Why?
+
+If an error occurs, the pending promise is not resolved and the server does not serve a response.
+In order to make sure that the end user receives an error response, the asyncWrapper functions as middleware
+that wraps our controllers and sends a response when an error is triggered. The asyncWrapper middleware helps prevent
+code repetition by providing a reocurring structure for the try/catch block. 
```

Comment:
Excellent answer;  I'm glad to see that you touched on functionality AND the purpose of readability/eliminating code repetition.